Gruntfile.js

```diff
@@ -1,33 +1,8 @@
-function encodingMiddleware (request, response, next) {
-  const URL = require('url').URL
-  var url = new URL(request.url, 'http://localhost')
-
-  if (url.pathname !== '/encoding') {
-    next()
-    return
-  }
-
-  var cookieName = url.searchParams.get('name')
-  var cookieValue = url.searchParams.get('value')
-
-  response.setHeader('content-type', 'application/json')
-  response.end(
-    JSON.stringify({
-      name: cookieName,
-      value: cookieValue
-    })
-  )
-}
-
 const config = {
   qunit: {
     all: {
       options: {
-        urls: [
-          'http://127.0.0.1:9998/',
-          'http://127.0.0.1:9998/module.html',
-          'http://127.0.0.1:9998/encoding.html?integration_baseurl=http://127.0.0.1:9998/'
```

Comment:
Need also to remove [this part](https://github.com/js-cookie/js-cookie/blob/d727ba6ceffb101e300a38b4eb06bcca4ac571d1/CONTRIBUTING.md#integration-with-server-side) from the README which documents the `?integration_baseurl`.